docs/deployment/release.md

```diff
@@ -87,7 +87,7 @@ git checkout prod
 
 git reset --hard origin/prod
 
-git tag YYYY.0M.R
+git tag -a YYYY.0M.R
 
```

Comment:
```suggestion
```

Git will open your default text editor and prompt you for the tag annotation. For the tag annotation, use the title of the `release`-tagged Issue that kicked off the release. Finally, after closing the text editor:

```bash
```

Reply:
Good idea to add more details on this! I see what you mean with the raw text. I'll make that change

Reply:
Done in https://github.com/cal-itp/benefits/pull/2104/commits/64c404b89579b1d43e6a4e48a64c03b17ccce4c2

Reply:
Looks good!